### ISSUE TITLE
Add support to web interface to show activations

### ISSUE THREAD

**Issue by dkwiebe** [cloudveiltech/CitadelManager#21]
Add support to the web interface to show activations per user.

**Comment by dkwiebe** [comment]
The database stores activations in app_user_activations that are linked to users.  When we double click on a user in the web gui I want the page that comes up to show the list of activations for the user.

**Comment by bluesven869** [comment]
![user activations](https://user-images.githubusercontent.com/23646985/32120206-154627be-bb2e-11e7-877d-2ad564209067.png)

**Comment by TechnikEmpire** [comment]
Wouldn't it complete another outstanding feature (have to look) to show last-use timestamp and have a delete button in here?

**Comment by bluesven869** [comment]
Okay, I will add `delete functionality`

**Comment by dkwiebe** [comment]
I'm going to add a few details on this one as the delete functionality is
tied in with another issue.

**Comment by TechnikEmpire** [comment]
My bad @bluesven869 I should have tagged @dkwiebe. He's the boss, I wasn't saying to add it was just trying to raise the suggestion to him.

**Comment by dkwiebe** [comment]
@bluesven869 Please add the last seen date to the table of installations.  

@TechnikEmpire Thanks for mentioning the deletes and thanks for commenting.  👍   I've added #32 to take care of deletes as it depends on #29 to actually sign out the user.

**Comment by dkwiebe** [comment]
@bluesven869 Can you double check this?  It was working before but is no longer showing any activations.

![image](https://user-images.githubusercontent.com/1034679/33237422-e133ade2-d230-11e7-9d53-e4ea14a161f8.png)

**Comment by dkwiebe** [comment]
@bluesven869 Please disregard that question.  I've taken care of it.
